src/Facebook/GraphNodes/GraphAlbum.php

```diff
@@ -110,9 +100,9 @@ public function getDescription()
     }
 
     /**
-     * Returns profile that created the album.
+     * Returns the profile that created the album.
      *
-     * @return GraphUser|null
+     * @return GraphNode|null
```

Comment:
Gotcha. Well this would be a BC (see the failing test). So we probably need to figure a way to support these changes from Graph without breaking the SDK. @yguedidi Any thoughts?